test/data/import/old/1.6.0.json

```diff
@@ -16,22 +16,23 @@
                     "documentation": "",
                     "method": "get",
                     "endpoint": "answer",
+                    "randomResponse": false,
```

Comment:
Please do not modify files in `test/data/migrations`, `test/data/import/new` and `test/data/import/old`